### DISCUSSION TITLE
Streaming simulator logs to terminal

### DISCUSSION THREAD

**Discussion by wojciech-kulik** [wojciech-kulik/xcodebuild.nvim#24] [Tips & Tricks]
This trick is especially useful if you are a tmux user. 

All logs are streamed to `console` window from `dap-ui-nvim` plugin, but they are also saved to a file. So you can just run one command to see logs live in your terminal.

```bash
tail -f .nvim/xcodebuild/app_logs.log
```

This way you can stream logs directly to another tmux window.

**Comment by guitaripod** [comment]
Can I just split the terminal and put this on the other instance? Tmux is just the better version of that, or?

**Reply by wojciech-kulik** [reply]
Yes, you can use splitting or whatever you like. It's just a bash command, you can run it from any place.

**Reply by guitaripod** [reply]
I ran the code in a sim and ran this in a different terminal window. It outputs some stuff when the build completes, but it's not outputting the runtime

**Reply by wojciech-kulik** [reply]
Oh, the filename was incorrect. It should be `app_logs.log`. I updated it :)

**Reply by guitaripod** [reply]
Might be some issue with the work repo and the dependency, but this is all I get

```
[CoreBluetoothMock] API MISUSE: <CoreBluetoothMock.CBMCentralManagerMock: 0x600001873940> can only accept this command while in the powered on state

**Reply by guitaripod** [reply]
It works with other projects, tho :)

**Comment by wojciech-kulik** [comment]
Moved to Wiki.
